assets/js/default-calendar.js

```diff
@@ -2,453 +2,432 @@ import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
```

Comment:
@Akhill2020 this file contains changes, rest are either autogenerated files or just for version bump. Please review with "Hide Whitespace" turned on.